futurex_openedx_extensions/dashboard/urls.py

```diff
@@ -28,5 +31,10 @@
     ),
     re_path(r'^api/fx/statistics/v1/course_statuses/$', views.CourseStatusesView.as_view(), name='course-statuses'),
     re_path(r'^api/fx/statistics/v1/total_counts/$', views.TotalCountsView.as_view(), name='total-counts'),
+    re_path(
+        fr'^api/fx/query/v1/(?P<scope>{QUERY_ALLOWED_SCOPES}+)/(?P<slug>{CLICKHOUSE_QUERY_SLUG_PATTERN})/$',
```

Comment:
The plus sign here makes no sense to me.

Would you mind sharing why we'd need the scope to be `platform` or `platformmmmm`?

```suggestion
        fr'^api/fx/query/v1/(?P<scope>{QUERY_ALLOWED_SCOPES})/(?P<slug>{CLICKHOUSE_QUERY_SLUG_PATTERN})/$',
```

It only makes sense as a plain regexp e.g. `[a-z0-9]+` which would match all of `omar`, `omarrrrrrr` and `omarrrr0121212`

Reply:
great catch, thank you!